### PR TITLE
Link to Informal and the Vienna Business Agency

### PR DESCRIPTION
**Pull request by shonfeder**
Calling out the orgs funding this work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,12 @@ Related reports and publications can be found at the
 
 [TLA+]: http://lamport.azurewebsites.net/tla/tla.html
 [Microsoft Z3]: https://github.com/Z3Prover/z3
+---
+
+Apalache is developed at [Informal Systems](https://informal.systems/).
+
+With additional funding from [![Vienna Business Agency Logo](TODO)](https://viennabusinessagency.at/).
```
Comment (shonfeder) [inline README.md:102]: @konnov do you know what logo we're meant to use here?

**Review by konnov — APPROVED** [review]
Thanks for taking care of it!

**Comment by shonfeder** [comment]
@konnov It's not taken care of because I don't which logo we are mean to use, so there's not actually a link to that. Do you know what's required on this front?

File: README.md (hunk 1 above)
Comment (konnov) [reply README.md:102]: The only logo I have found is this one: https://viennabusinessagency.at/fileadmin/user_upload/Ad_Logo/Logo_Dt__Wirtschaftsagentur_StadtWien.zip

Is it too large?

Comment (shonfeder) [reply README.md:102]: That'll work! Thanks :)

**Comment by konnov** [comment]
Do you know how shall one mention the past sponsors? https://wwtf.at/ was supporting us for 4 years. So we definitely have to mention them.

**Comment by shonfeder** [comment]
No, I don't know how they should be mentioned. Do they have guidelines on it?

**Comment by shonfeder** [comment]
I added mention of past funding from the WWFT.

Rendered file here: https://github.com/informalsystems/apalache/blob/958d0256b1e3a4b0bc5d2f009899d66fe3a7e8f6/README.md

**Comment by shonfeder** [comment]
Doh! I'd targetd this to master for some reason. Will resolve merge conflicts.

**Comment by konnov** [comment]
> Doh! I'd targetd this to master for some reason. Will resolve merge conflicts.

That must be pain. We should probably come back to the question of having `master` at all. It does not seem to be useful anymore.

**Pull request merged**